### PR TITLE
Add form_theme version of form.fixers

### PR DESCRIPTION
**Pull request by HedicGuibert**
Instead of accessing form variables to render the fixers checkboxes, it may be better to have a dedicated form theme, so here it is.

I also removed a few forgotten useless CSS lines and add hover behaviour on all links - I forgot that, too.

### CODE REVIEW

File: templates/main/fixers.html.twig [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <input type="checkbox" class="js-checkbox u-my(half) u-as(center)" id="typo_fixers_{{ name }}" value="{{ value }}" name="typo_fixer[fixers][]"{% if checked %} checked="checked"{% endif %}>
+{% endblock %}
+
+{% block form_errors %}
+    {%- if errors is defined -%}
+        {%- for error in errors -%}
+            <div class="error-message u-c(error) u-fz(h5) u-bor(error) u-d(inline-block) u-py(half) u-px(base) u-br(5) u-mb(double)">{{ error.message }}</div>
+        {%- endfor -%}
+    {%- endif %}
+{% endblock %}
```
Comment (lyrixx) [inline templates/main/fixers.html.twig:33]: I think this is not required, since there is already a form_theme on the global form.

**Review by lyrixx — COMMENTED** [review]
Except one little comment, I like it 👍🏼 👏🏼

File: templates/main/fixers.html.twig (hunk 1 above)
Comment (HedicGuibert) [reply templates/main/fixers.html.twig:33]: I may have done it wrong but if I leave the block empty the errors don't show up anymore

File: templates/homepage/homepage.html.twig [hunk 2]
```diff
@@ -1,15 +1,6 @@
...
-{% form_theme form.fixers 'homepage/fixers.html.twig' %}
-
-{% block form_errors -%}
-  {%- if errors is defined -%}
-    {%- for error in errors -%}
-      <div class="error-message u-c(error) u-fz(h5) u-bor(error) u-d(inline-block) u-py(half) u-px(base) u-br(5) u-mb(double)">{{ error.message }}</div>
-    {%- endfor -%}
-  {%- endif %}
-{%- endblock form_errors %}
+{% form_theme form 'homepage/fixers.html.twig' %}
```
Comment (lyrixx) [inline templates/homepage/homepage.html.twig:3]: I'm not sure it deserves another file. Could you move everything here ?

**Review by lyrixx — APPROVED** [review]
Good to me 👍🏼

**Comment by lyrixx** [comment]
@damienalexandre I think you can merge this PR, and merge `next` in `master`, and deploy 🎉

**Review by damienalexandre — APPROVED** [review]

**Pull request merged**